test/Microsoft.AspNet.PlatformHandler.FunctionalTests/HelloWorldTest.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Server.Testing;
+using Microsoft.AspNet.Testing.xunit;
+using Microsoft.Framework.Logging;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Loader.IIS.FunctionalTests
+{
+    // Uses ports ranging 5061 - 5069.
+    public class HelloWorldTests
+    {
+        [ConditionalTheory]
+        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x86, "http://localhost:5061/", ServerType.Kestrel)]
+        [InlineData(RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:5062/", ServerType.Kestrel)]
+        [InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x86, "http://localhost:5063/", ServerType.Kestrel)]
+        [InlineData(RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:5064/", ServerType.Kestrel)]
+        public Task HelloWorld_IISExpress(RuntimeFlavor runtimeFlavor, RuntimeArchitecture architecture, string applicationBaseUrl, ServerType delegateServer)
```

Comment:
Just realize that this is not at the sample repo as __DNX__ so the test infrastructure is out of reach.

It would be worth to consider to pull our test infrastructure to a independent package so as to be consumed across repo. @davidfowl @JunTaoLuo